README.md

```diff
@@ -8,6 +8,7 @@ The web application is for education purpose only and is not meant for streaming
 
 - [Team](#team)
 - [Tech stack](#tech-stack)
+- [API Documentation](https://app.swaggerhub.com/apis/hypertubeHive/Hypertube/1.0.0)
```

Comment:
Good job!
maybe use this link? https://app.swaggerhub.com/apis-docs/hypertubeHive/Hypertube/1.0.0 to see the docs, without the possibility to edit.

Reply:
good point! changed it 👍